src/ValueObjects/Video.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\SearchV3\ValueObjects;
+
+use JMS\Serializer\Annotation\Type;
+
+final class Video
```

Comment:
Can we strong type the properties on this VO as well? I think it might remove the need for the @Type annotation

Reply:
Yes, but that will have to wait for the vagrant PHP 7.4 upgrade for project-aanvraag which needs videos for a new feature.

Reply:
I will do that on a new major release of this library and also work with DTOs and reduce the code base.